dallinger/recruiters/mturk/__init__.py

```diff
@@ -0,0 +1,8 @@
+"""
+Each recruiter consists of two parts: a service (doing the API calls) and the actual recruiter (doing the logic).
+"""
+
+from dallinger.recruiters.mturk.messages import *  # noqa # pylint: disable=unused-import
```

Comment:
I wonder about instead importing the module names and using them as a namespace. This would be clearer, since you could see where names come from:
```
from dallinger.recruiters.mturk import messages
from dallinger.recruiters.mturk import qualifications

# later...
messages.something(foo)
qualifications.something(bar)
```

Reply:
Great suggestion!